src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/GeneratedDocumentContainer.cs

```diff
@@ -16,12 +16,7 @@ internal class GeneratedDocumentContainer
         public event EventHandler<TextChangeEventArgs> GeneratedCSharpChanged;
         public event EventHandler<TextChangeEventArgs> GeneratedHtmlChanged;
 
-        private SourceText _source;
         private VersionStamp? _inputVersion;
-        private VersionStamp? _outputCSharpVersion;
-        private VersionStamp? _outputHtmlVersion;
-        private RazorCSharpDocument _outputCSharp;
-        private RazorHtmlDocument _outputHtml;
```

Comment:
At some point we should probably revisit all of these. I'm sure they're responsible for all sorts of chaos in our system. Aka, the fact we even have a generated document container seems like a pit of failure because every snapshot contributes to it